examples/jsm/loaders/FBXLoader.js

```diff
@@ -169,8 +169,17 @@ class FBXLoader extends Loader {
 		}
 
 		// console.log( fbxTree );
+		let textureLoader = this.manager.getHandler( this.resourcePath || path );
```

Comment:
The path/resource path does not contain the file extensions. It's just the base path.